docs/MVS_Outputs.rst

```diff
@@ -155,6 +155,47 @@ To measure this, we propose to compare the energy flows in between the sectors t
 
 ** The content of this section was copied from the conference paper handed in to CIRED 2020**
 
+Self-consumption (SC)
+ ################
+
+The self-consumption describes the fraction of all renewable energy that is consumed by the system itself.
+
+.. math::
+        SC &=\frac{\sum_{i,j} {E_{generation} (i,j) \cdot w_i} - E_{gridfeedin}(i,j) \cdot w_i}{\sum_{i,j} {E_{generation} (i,j) \cdot w_i}}
+
+        \text{with~} &i \epsilon \text{[PV,Geothermal,…]}
+
+        &j \epsilon \text{[generation assets 1,2,…]}
+
+[1] https://www.sciencedirect.com/science/article/pii/S0960148119315216
+[2] https://www.iip.kit.edu/downloads/McKennaetal_paper_full.pdf
+
+Self-sufficiency (SS)
+###############
+
+The self sufficiency describes the fraction of the total demand that can be covered by renewable generated energy see [1] and [2]
+
+.. math::
+        SS &=\frac{\sum_{i,j} {E_{generation} (i,j) \cdot w_i} - E_{gridfeedin}(i,j) \cdot w_i}{\sum_i {E_{demand} (i) \cdot w_i}}
+
+        \text{with~} &i \epsilon \text{[PV,Geothermal,…]}
+
+        &j \epsilon \text{[generation assets 1,2,…]}
+
+
+Electrical Autonomy (EA)
```

Comment:
I'm not sure about the wording here, as it's not only about electricity but also other sectors, I assume. What about "degree of autonomy" if it's not already defined in literature like our SS above. Or "Net-Autonomy"?

Reply:
changed it already